i18n/bundle/bundle_test.go

```diff
@@ -33,6 +34,28 @@ func TestMustTfunc(t *testing.T) {
 	New().MustTfunc("invalid")
 }
 
+func TestLanguageTagsAndTranslationIDs(t *testing.T) {
+	b := New()
+	translationID := "translation_id"
+	englishLanguage := languageWithTag("en-US")
+	frenchLanguage := languageWithTag("fr-FR")
+	spanishLanguage := languageWithTag("es")
+	addFakeTranslation(t, b, englishLanguage, "English"+translationID)
+	addFakeTranslation(t, b, frenchLanguage, translationID)
+	addFakeTranslation(t, b, spanishLanguage, translationID)
+
+	tags := b.LanguageTags()
+	sort.Strings(tags)
+	compareTo := []string{englishLanguage.Tag, spanishLanguage.Tag, frenchLanguage.Tag}
+	if !reflect.DeepEqual(tags, compareTo) {
+		t.Error("Problem encountered with LanguageTags.")
+	}
+
+	if b.LanguageTranslationIDs(englishLanguage.Tag)[0] != "English"+translationID {
```

Comment:
I think you should do the same thing you did above and use an array instead of just checking the first element in the array. That way we also test there aren't additional elements we aren't expecting.